api-examples/__includes/async_mysql_require_env.inc.php

```diff
@@ -1,14 +1,19 @@
-<?hh // partial
+<?hh
 
 namespace Hack\UserDocumentation\API\Examples\AsyncMysql;
 
-require __DIR__."/async_mysql_connect.inc.php";
+use Hack\UserDocumentation\API\Examples\AsyncMysql\ConnectionInfo as CI;
 
-use \Hack\UserDocumentation\API\Examples\AsyncMysql\ConnectionInfo as CI;
+<<__EntryPoint>>
+function async_mysql_skipif(): void {
+  require __DIR__."/async_mysql_connect.inc.php";
 
-if (!extension_loaded('mysql') || !function_exists('mysqli_connect')) {
-  die('Skip');
-}
-if (!mysqli_connect(CI::$host, CI::$user, CI::$passwd, CI::$db, CI::$port)) {
-  die('Skip');
+  $pool = new \AsyncMysqlConnectionPool(darray[]);
+  try {
+    \HH\Asio\join(
```

Comment:
Given callers/includes need modifying anyway, is it practical t oturn this into an async function and await, moving HH\Asio\join() out a level if needed? It'd be good to have no hh\asio\join in these at all